src/navigation/_AppSettingsProvider.jsx

```diff
@@ -58,8 +64,14 @@ const AppSettingsProvider = ({ settingsConfig, children }) => {
       }
     };
 
+    /* Use default locales or fall back on user configurations configurations.
+     * terra-toolkit v5.21.0 provides configurations from terraI18n, so this reduces the number of places users need to define their locales.
+     */
+    const locales = typeof TERRA_AGGREGATED_LOCALES === 'object' ? TERRA_AGGREGATED_LOCALES : settingsConfig.locales;
```

Comment:
does this fallback correctly if we remove the value in the [config here](https://github.com/cerner/terra-dev-site/pull/270/files#diff-4e555628656c4d992b59e6e269212191L79)?

I'm assuming we wouldn't hit this often, if ever if they use the terra-dev-site webpack config

Reply:
Right now, if the global is undefined, it will fallback to whatever the user has defined in their `settings.config.json` file. If they didn't define locales in that file either, it will default to an empty array. We could also either fallback to the full array of default values or just an array with 'en' in it depending on what you think is best.

Reply:
I think we should just fall back to `['en']` if we're saying that the locales config has been deprecated

Reply:
Just pushed a change to fall back to just english [here](https://github.com/cerner/terra-dev-site/blob/03979319ba8e562fd7609d917d177337cb6da055/src/navigation/_AppSettingsProvider.jsx#L70).